charts/k8ssandra/templates/cassandra/cassdc.yaml

```diff
@@ -114,6 +114,17 @@ spec:
   podTemplateSpec:
     spec:
       initContainers:
+      - name: base-config-init
+        image: {{ default (get .Values.cassandra.versionImageMap .Values.cassandra.version) .Values.cassandra.image }}
```

Comment:
We use this template in a couple places now to get the Cassandra image. It is a bit of a mouthful. Can you create a helper template for this in _helpers.tpl? Take a look at https://github.com/k8ssandra/k8ssandra/blob/main/charts/k8ssandra/templates/_helpers.tpl#L12 for an example.

Reply:
will do